docs/DeveloperGuide.md

```diff
@@ -1220,29 +1220,202 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+<div style="page-break-after: always;"></div>
+
+### Adding a person
```

Comment:
We're missing out a few of the test cases here like editPerson. Will approve for the draft but please add them in soon.